### ISSUE TITLE
Add metadata external link to Library Import

### ISSUE THREAD

**Issue by charredchar** [Sonarr/Sonarr#4996]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe

I often will already have a full series that is not sorted/cleaned where I will use the Library Import feature to get it into Sonarr then have it automatically cleaned up, works great! One big issue is with Anime but it can happen with regular shows as well, what happens when you have multiple results with the same/similar names but you're unsure if it is right?

### Describe the solution you'd like

For the list of returned Series during the import process it would be nice to have an icon next to each series in the dropdown to open the metadata page (TheTVDB, I assume) to ensure it is the one you actually want. This has already been implemented with the Add New option and is exactly what I would want just for the Library Import.

### Describe alternatives you've considered

Personally, I feel a link icon for each returned result would be best but at a minimum having one for the selected result, next to the drop down for instance, would also work. This would just be more tedious if the one you select wasn't it.

### Anything else?

Sick* does similar when adding shows, each result lets you click the title to open the link to your metadata indexer of choice.

**Comment by charredchar** [comment]
I am currently importing a bunch of shows which Sick* use to manage which is why this issue is on my mind now. I have a very good example of how a link button would greatly help. I don't remember what year Heroes aired and in the list it is the only detail (minus channel, which isn't always present and assuming you even remember that on a 15 year old series) setting each result apart.
![image](https://user-images.githubusercontent.com/1829271/164280785-fee0191b-e433-46cf-90ae-4cee4421cb72.png)

**Comment by charredchar** [comment]
Thanks for the details! These files are not directly from a download folder and have technically been organized once before by a different program and even include NFO files for most. But since these did not match the scheme Sonarr will use I found it (generally) best to Import Library then Mass Edit to move and sort files. This is nearly 300 shows! Adding each by hand then pointing to a folder to pull from and hope it didn't mix anything up made me want to check each as Library Import allows. Honestly, I just finished and it'll be a very long time before needing the tool again (hopefully) but my thought was for next time or for anyone else this feature would help a lot.
